superset/assets/src/explore/components/AdhocFilterEditPopover.jsx

```diff
@@ -78,6 +79,13 @@ export default class AdhocFilterEditPopover extends React.Component {
     document.removeEventListener('mousemove', this.onMouseMove);
   }
 
+  adjustHeight(heightDifference) {
+    this.setState(state => ({
+      ...state,
+      height: state.height + heightDifference,
```

Comment:
Just curious, is there a specific reason to add the height difference instead of just strictly setting the new height?

Reply:
FWIW the `re-resizable` library I'm using for the new dashboard passes a delta.

Reply:
The big issue is that there are two things that can resize the popover ->

1) the user dragging the box

2) the input component expanding

tracking deltas just simplified dealing with both of those